src/components/HeadCellMenu/MenuList/MenuGroup.tsx

```diff
@@ -50,6 +50,7 @@ const MenuGroupItems = ({
   const anchorRef = useRef<HTMLDivElement | null>(null);
 
   const handleOnClick = async (evt: React.MouseEvent<HTMLLIElement, MouseEvent>) => {
+    if (evt.type === 'keyup') return;
```

Comment:
I think this needs a breif comment. It doesn't make sense that we need to catch a key event in here